src/languages/kotlin.js

```diff
@@ -0,0 +1,106 @@
+/*
+Language: Kotlin
+Author: Sergey Mashkov <cy6erGn0m@gmail.com>
+Category: common
```

Comment:
"common" is the only category that at this moment is special: it defines a small subset of very commonly used languages that goes into a pre-built script we host on CDNs. So it's best not to use it for new languages. Here it can simply be dropped unless it fits well into any of the existing categories (https://highlightjs.org/static/demo/).